app/pages/otherOfficials/IsOtherOfficialsPreviousAddressPage.scala

```diff
@@ -16,12 +16,22 @@
 
 package pages.otherOfficials
 
+import models.UserAnswers
 import pages.QuestionPage
+import pages.addressLookup.OtherOfficialPreviousAddressLookupPage
 import play.api.libs.json.JsPath
 
+import scala.util.Try
+
 case class IsOtherOfficialsPreviousAddressPage(index:Int) extends QuestionPage[Boolean] {
 
   override def path: JsPath =  OtherOfficialsId(index).path \ toString
 
   override lazy val toString: String = "isOfficialsPreviousAddress"
+
+  override def cleanup(value: Option[Boolean], userAnswers: UserAnswers): Try[UserAnswers] =
+    value match {
+      case Some(false)  => userAnswers.remove(OtherOfficialPreviousAddressLookupPage(index))
+      case _ => Try(userAnswers)
```

Comment:
any particular reason you made this `Try(userAnswers)` instead of `super.cleanup(value, userAnswers)` (which is equivalent to `Success(userAnswers)`)?